### PR TITLE
Add News page and some initial posts

### LINKED ISSUE #9: Add News section
- Listing page
- Detail page
- News item in nav

### PR DESCRIPTION
**Pull request by rebkwok**
Add the news page, plus nav links.

Also adds 4 initial posts.  Only the first is intended to be published immediately, but all are currently set as published so they can be reviewed in the deploy preview link.

1) Announcing DJE in Edinburgh (publish now)
2) CFP coming soon - announcing CFP open date - publish tomorrow if possible (6th Jan)
3) CFP open - publish 16th Jan
4) Why/how to speak - publish 31st jan during CFP to encourage submissions

Note that some links will 404 atm because they're dependent on other PRs

Closes #9

### CODE REVIEW

**Comment by rebkwok** [comment]
Made this PR draft as a reminder to mark the future posts as unpublished before merging

**Comment by thibaudcolas** [comment]
👍 just starting to review this – I’ll change how the Netlify build works so we display drafts there only.

File: _layouts/post-list.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {% for post in site.posts %}
   {% unless post.hidden or post.category == 'talk' or post.category == 'tutorial' %}
   <article class="post-item">
-    <time class="post-meta">{{ post.date | date: "%b %-d, %Y" }}</time>
+    <time class="post-meta">{{ post.date | date: "%d %b %Y" }}</time>
```
Comment (thibaudcolas) [inline _layouts/post-list.html:9]: It looks like the post list on the homepage needs updating as well (and fixing).

File: _posts/2023-01-01-announcing-djangocon-europe-2023.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+post_photo_alt: The view of Edinburgh from Calton Hill
+post_photo_url: /static/img/locations/calton-hill.jpg
+title: DjangoCon Europe is coming to Edinburgh!
+---
+
+Hello, everyone!  This year, DjangoCon Europe is coming to Edinburgh, Scotland!
+
+We are very excited to announce that the website for DjangoCon Europe 2023 is [live](https://2023.djangocon.eu).
+
+Many thanks to [TorchBox](https://torchbox.com/) for the website design!
```
Comment (thibaudcolas) [inline _posts/2023-01-01-announcing-djangocon-europe-2023.md:16]:
```suggestion

Many thanks to [Torchbox](https://torchbox.com/) for the website design!
```

File: _posts/2023-01-01-announcing-djangocon-europe-2023.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The conference will be from Monday 29th May to Friday 2nd June 2023.
+
+- 29-31 May: Talks, Tutorials and Workshops
+- 1-2 June: Sprints
+
+Talks, tutorials and workshops will be held at the [Assembly Rooms](/venues/#talks), in 
+the heart of Edinburgh's city centre. Sprints will be held at the 
+[Biscuit Factory](/venues/#sprints) in Leith.
```
Comment (thibaudcolas) [inline _posts/2023-01-01-announcing-djangocon-europe-2023.md:25]: I see we have an issue with the site stylesheet adapting to those anchor links. Will change the CSS.

File: _posts/2023-01-01-announcing-djangocon-europe-2023.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+We'll also have a social event at an amazing venue, [The Caves](/venues/#social)!
+
+The Call for Proposals and ticket sales will be open soon.  In the meantime, you can get an
```
Comment (thibaudcolas) [inline _posts/2023-01-01-announcing-djangocon-europe-2023.md:29]:
```suggestion
The Call for Proposals and ticket sales will be open soon. In the meantime, you can get an
```

File: _posts/2023-01-01-announcing-djangocon-europe-2023.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+insight into the city [here](/location).
```
Comment (thibaudcolas) [inline _posts/2023-01-01-announcing-djangocon-europe-2023.md:30]:
```suggestion
insight into the city on our [Location page](/location). 
```

File: _posts/2023-01-01-announcing-djangocon-europe-2023.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+Talks, tutorials and workshops will be held at the [Assembly Rooms](/venues/#talks), in 
+the heart of Edinburgh's city centre. Sprints will be held at the 
+[Biscuit Factory](/venues/#sprints) in Leith.
+
+We'll also have a social event at an amazing venue, [The Caves](/venues/#social)!
+
+The Call for Proposals and ticket sales will be open soon.  In the meantime, you can get an
+insight into the city [here](/location). 
+
+We hope we'll see you all at DjangoCon Europe 2023, and don't forget to follow us [@DjangoConEurope](https://twitter.com/djangoconeurope) on Twitter
```
Comment (thibaudcolas) [inline _posts/2023-01-01-announcing-djangocon-europe-2023.md:32]:
```suggestion
We hope we'll see you all at DjangoCon Europe 2023, and don't forget to follow us [@{{ site.twitter.username }}](https://twitter.com/{{ site.twitter.username }}) on Twitter.
```

File: _posts/2023-01-06-call-for-proposals-coming-soon.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+- Django internals and challenges in modern web development.
+- Wild ideas, clever hacks, surprising or cool use cases.
+- Improving Django and Python developers’ lives.
+- Pushing Django to its limits
+- Fundamentals, in a modern light.
+- The Django and Python community, culture, history, past, present & future, the why the who and the what of it all
+- Security
+- Whatever you deem appropriate – it's your conference, after all!
+
+Take a look at our [information for speakers](/speaking), and get ready to submit your proposal!
```
Comment (thibaudcolas) [inline _posts/2023-01-06-call-for-proposals-coming-soon.md:26]: This is currently a 404.

File: _posts/2023-01-16-announcing-call-for-proposals.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+We welcome first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it!
+
+Plus there are perks! Presenters get free admission to DjangoCon Europe, including admission to the social event! 
+
+For more information on talk and tutorial formats, please check out our [speaker information page](/speaking/).
+
+We want everyone attending DjangoCon Europe to feel safe, welcome, and included. To that end, we have a [Code of Conduct](/conduct/) for all speakers and attendees.
+
+If you have questions feel free [to contact the content team]((mailto:content@djangocon.eu)).
```
Comment (thibaudcolas) [inline _posts/2023-01-16-announcing-call-for-proposals.md:24]:
```suggestion
If you have questions feel free [to contact the content team](mailto:{{ site.content_email }}).
```

File: _posts/2023-01-31-why-speak-at-djangocon-europe.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+Now that we've convinced you that you should speak, watch [Mark's talk](https://youtu.be/4rsL974kwsE) on how to get on the DjangoCon Europe stage and what do you when you get there!
+
+<div class="responsive-embed widescreen">
+  <iframe width="917" height="516" src="https://www.youtube.com/embed/4rsL974kwsE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+</div>
+
+The [Call for Proposals](https://pretalx.com/djangocon-europe-2023/cfp) is open until 26th February 2023 [AoE](https://time.is/compare/2359_26_February_2023_in_Anywhere_on_Earth).
+
+
+If you have questions, get in touch with the content team: [content@djangocon.eu](mailto:content@djangocon.eu).
```
Comment (thibaudcolas) [inline _posts/2023-01-31-why-speak-at-djangocon-europe.md:32]:
```suggestion
If you have questions, get in touch with the content team: [{{ site.content_email }}](mailto:{{ site.content_email }}).
```

**Review by thibaudcolas — APPROVED** [review]
Looking great! I’ll move the draft posts to `_drafts` and merge.

I’ve updated the blog posts to use the variables we have defined in `_config.yml`, not too sure whether that was worthwhile but we’ll see if we ever need to change any.

**Pull request merged**